### PR TITLE
Refactoor : Orchs

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

File: asabiris/orchestration/sendemail.py [hunk 1]
```diff
@@ -1,228 +1,83 @@
...
+class SendEmailOrchestrator:
+
+    def __init__(self, app):
+        self.services = {
+            name: app.get_service(name) for name in [
+                "JinjaService", "HtmlToPdfService", "MarkdownToHTMLService", "SmtpService"
+            ]
+        }
+
+    async def send_email(self, email_to: List[str], body_template: str, **kwargs):
```
Comment (ateska) [inline asabiris/orchestration/sendemail.py:24]: Explicit parameters (not *kwargs) are better b/c of the documentation.

File: asabiris/orchestration/sendemail.py [hunk 2]
```diff
@@ -1,228 +1,83 @@
...
+            jinja_output = await self.services['JinjaService'].format(template, params)
+            ext = os.path.splitext(template)[1]
+            if ext not in ['.html', '.md']:
+                raise FormatError(format=ext)
+            return {
+                '.html': utils.find_subject_in_html,
+                '.md': lambda x: (self.services['MarkdownToHTMLService'].format(utils.find_subject_in_md(x)[0]), utils.find_subject_in_md(x)[1])
+            }[ext](jinja_output)
+        except (jinja2.exceptions.TemplateError, Exception) as e:
+            if asab.Config.get("jinja", "failsafe"):
```
Comment (ateska) [inline asabiris/orchestration/sendemail.py:50]: Failsafe should not be configurable, it should be always enabled.
But thru exception.

File: asabiris/orchestration/sendemail.py [hunk 3]
```diff
@@ -1,228 +1,83 @@
...
+                    "Hello!<br><br>"
+                    "We encountered an issue while processing your request. "
+                    "Please review your input and try again.<br><br>"
+                    "Thank you!<br><br>Error Details:<br><pre style='font-family: monospace;'>Timestamp: {}\nRecipients: {}\nError Message: {}\n</pre>"
+                    "<br>Best regards,<br>LogMan.io".format(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"), ', '.join(email_to), str(e))
+                )
+                return error_message, "Error Processing Request"
+            raise
+
+    def _process_attachments(self, attachments: List[Dict], email_to: List[str]) -> List[Tuple[Union[str, bytes], str, str]]:
```
Comment (ateska) [inline asabiris/orchestration/sendemail.py:61]: `email_to` is unnecessary parameter.

**Review by ateska — CHANGES_REQUESTED** [review]

File: asabiris/orchestration/sendemail.py (hunk 3 above)
Comment (mithunbharadwaj) [reply asabiris/orchestration/sendemail.py:70]: @ateska  **email_to** ,  it also serves a secondary purpose: in the event of a rendering error and when the failsafe mode is activated, the email_to parameter provides the recipient details in the error message to inform them about the encountered issue.

**Comment by ateska** [comment]
Ok, let me know when ready for review & merging.

**Pull request merged**